Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,110 @@
+
+This sample uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the
+ LOCAL CPC (default) or the *CPCname* provided
+- List the LPARs on the designated CPC and query the LPAR status.
+- For each LPAR matching the requested status (default is operating), access the image activation profile with the same name as the LPAR to rerieve the 
+following information:
+     - crypto-activity-cpu-counter-authorization-control
+     - assigned-crypto-domains
+     - assigned-cryptos
+- <b>Note: </b>
+ The crypto attributes are available on z16 or higher processors only.
+
+## System Prep work
+- Store RXCRYPT1 into a data set
+- Ensure your z/OS user ID has at least READ access to the following FACILITY Class Profiles
+    - HWI.TARGET.netid.nau
+    - HWI.TARGET.netid.nau.imagename
+
+    <p>where netid.nau represents the 3– to 17– character SNA name of the particular CPC whose LPARs are being queried and imagename represents the 1– to 8- character LPAR name, or * to represent all of the LPARs available on that CPC </p>
+
+- Allocate a partitioned data set that can hold the generated content
+   - the longest line generated depends on how many assigned-crypto-domains and assigned-cryptos are found
+   - minimum data set characteristics
+     - RECFM: VB
+     - LRECL: 800
+
+## Invocation
+**Syntax**:
+```
+  RXCRYPT1 -D outputDataSet [-C CPCname] [-S Status] [-I] [-V]                                                
+ ```
+ where:
+  - *-D outputDataSet*
+      - **required**
+      - *outputDataSet* is the name of an existing partitioned data set
+      - if the query is successful, a member containing the crypto
+        information in a CSV format will be stored into the specified data set: *outputDataSet(memberName)* where the name of data set member is:
+        - LOCAL, if the LOCAL CPC was used
+        - CPCname, if a specific CPC was specified via the -C option
+  - *-C CPCname*
+      - **optional**
+      - *CPCname* is the name of the CPC which hosts the LPARs whose corresponding image activation profile crypto info will be retrieved
+      - **default if not provided:** LOCAL CPC
+  - *-S Status*
+      - **optional**
+      - *Status* represents the status of the LPAR and is an optional filter for the list of LPARs. The names associated with these LPARs is used as input for which image activation profiles need to be queried.
+      - Valid values:          
+        - operating - query profiles for activation image profiles
+        associated with active/operating LPARs  
+        - not-operating - query profiles for activation image profiles     associated with active/not operating LPARs                                                  
```

Comment:
looks like there are a few too many spaces between profiles and associated

Reply:
fixed